manifests/m/Microsoft/Azure/DataCLI/20.3.13/Microsoft.Azure.DataCLI.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.Azure.DataCLI
+PackageVersion: 20.3.13
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://www.microsoft.com/en-us/
+PublisherSupportUrl: https://support.microsoft.com/en-us
+PrivacyUrl: https://privacy.microsoft.com/en-us/privacystatement
+Author: Microsoft Corporation
+PackageName: Azure Data CLI
+PackageUrl: https://docs.microsoft.com/en-us/sql/azdata/install/deploy-install-azdata-installer?view=sql-server-ver15
```

```suggestion
PackageUrl: https://docs.microsoft.com/sql/azdata/install/deploy-install-azdata-installer?view=sql-server-ver15
```
nit: To let the webpage choose the locale